src/internal/data-service-api/data-service-dialects-api.ts

```diff
@@ -187,6 +203,10 @@ export interface DialectSummaryDto {
   readonly memberSummaries: MemberSummaryDto[];
 }
 
+export interface DialectsSummaryDto {
+  readonly unreadMessagesAmount: number;
```

Comment:
I think, `count` fits better than `amount` in this context, wdyt? same applies to data-service, I think

Reply:
True, agree.